dbclient/WorkspaceClient.py

```diff
@@ -330,8 +335,22 @@ def init_workspace_logfiles(self, workspace_log_file='user_workspace.log',
         if os.path.exists(libs_log):
             os.remove(libs_log)
 
-    def log_all_workspace_items(self, ws_path='/', workspace_log_file='user_workspace.log',
-                                libs_log_file='libraries.log', dir_log_file='user_dirs.log'):
+    def log_all_workspace_items_entry(self, ws_path='/', workspace_log_file='user_workspace.log', libs_log_file='libraries.log', dir_log_file='user_dirs.log'):
```

Comment:
Initiates the ThreadSafeWriters and pass it to `log_all_workspace_items` who actually do the work.